### ISSUE TITLE
Should "XHR Request" simply be referred to as "XHR"?

### ISSUE THREAD

**Issue by VirtualDOMinic** [foundersandcoders/api-workshop#35]
The extra "request" seems a bit redundant as the 'R' in "XHR" already stands for "request". Probably just me being a bit picky but...

**Comment by zurda** [comment]
I agree and happy to change all `XHR Requests` to ` XMLHttpRequest (XHR)`. I don't think it's being picky. For me having an acronym that doesn't make sense can really affect how well I remember what it means...
